convert2rhel/utils.py

```diff
@@ -718,16 +718,25 @@ def is_dir_empty(path):
             os.rmdir(path)
 
 
-def hide_secrets(args, secret_args=frozenset(("--password", "--activationkey", "--token", "-p", "-k"))):
```

Comment:
I've removed the "--token" because convert2rhel does not accept such an option and we don't expect it to accept it in the future (I don't know about any kind of token that could be used during the conversion).